arches/app/templates/views/components/search/search-export.htm

```diff
@@ -216,25 +216,16 @@
         <!-- /ko -->
     </div>
 
-    <!-- Celery Warning -->
-    <!-- ko let: {show_celery_warning: ((total() > {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD}}) && (celeryRunning() !== 'True') || (format() === 'html' && total() > {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD_HTML_FORMAT}} && celeryRunning() !== 'True'))} -->
-    <div 
-        id="export-warning"
```

Comment:
Seems like we should also remove the `aria-describedby` that's using this (and maybe the `ko let` comment?)